duckduckgo/iOS
# Accessibility localization items

Task/Issue URL: https://app.asana.com/0/72649045549333/1202621026047124/f
Tech Design URL:
CC: @mallexxx 

**Description**:
Add missing translation to localization strings

**Steps to test this PR**:
1. Change your device to a non-english supported language 
2. Turn on voice over on accessibility
3. Check Auto fill show/hide button 


**Device Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
